Add deny connection tracking for Flow Exporter (#2112)
antrea-io/antrea: docs/network-flow-visibility.md

flows.

Kubernetes information such as Node name, Pod name, Pod Namespace, Service name,
NetworkPolicy name and NetworkPolicy Namespace, is added to the flow records.
Network Policy Rule Action (Allow, Reject, Drop) is also supported for both

> Antrea-native NetworkPolicies and K8s NetworkPolicies For K8s NetworkPolicies,

Reviewer: ```suggestion
Antrea-native NetworkPolicies and K8s NetworkPolicies. For K8s NetworkPolicies,
```

... missing period

> connection dropped due to automatic isolation will be assigned with Drop action.

Reviewer: ```suggestion
connections dropped due to [isolated Pod behavior](https://kubernetes.io/docs/concepts/services-networking/network-policies/#isolated-and-non-isolated-pods) will be assigned the Drop action.
```

flows.

Kubernetes information such as Node name, Pod name, Pod Namespace, Service name,
NetworkPolicy name and NetworkPolicy Namespace, is added to the flow records.
Network Policy Rule Action (Allow, Reject, Drop) is also supported for both

> Antrea-native NetworkPolicies and K8s NetworkPolicies.For flow records that are

Author: Action value for connections dropped due to isolation will be `Drop` for K8s NP
